Makefile

```
@@ -22,20 +23,23 @@ console:
 
 build-latest:
 	$(MAKE) build TAG=latest \
+		PROTOCOL_VERSION_DEFAULT=19 \
```

Is this expected to be 19?

---

👀 No. No it is not. I was testing locally and accidentally pushed it. Thank you!
```suggestion
		PROTOCOL_VERSION_DEFAULT=20 \
```